Makefile

```diff
@@ -126,6 +126,9 @@ format: $(GOIMPORTS)
 $(GOX):
 	(cd /; GO111MODULE=on go get -u github.com/mitchellh/gox)
 
+$(GOLANGCI_LINT):
+	(cd /; GO111MODULE=on GOLANGCI_LINT_VERSION="1.21.0" ./.circleci/bootstrap.sh)
```

Comment:
Would it be possible to simply do the "go get" here? I don't think we should be calling out to scripts specific to circleci (i do realize that it is installing golangcilint)

Reply:
> 
> 
> Would it be possible to simply do the "go get" here? I don't think we should be calling out to scripts specific to circleci (i do realize that it is installing golangcilint)

It is not recommended to install via `go get` in the home page: https://github.com/golangci/golangci-lint#go

Reply:
@longkb I think that would probably be best, then in circle bootstrap we can call the new make target

Reply:
Done. I did it :) Please give me your comments if needed